packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -103,6 +103,10 @@ class CupertinoCheckbox extends StatefulWidget {
     this.tristate = false,
     required this.onChanged,
     this.activeColor,
+    @Deprecated(
+      'Use fillColor instead. '
+      'This feature was deprecated after v3.24.0-0.2.pre.'
+    )
```

Comment:
```suggestion
    @Deprecated(
      'Use fillColor instead. '
      `fillColor now manages the background color in all states. '
      'This feature was deprecated after v3.24.0-0.2.pre.'
    )
```
Prefer adding another line giving the reasoning.